util/util.go

```diff
@@ -600,3 +630,26 @@ func GetAppVersion(ctx *ankh.ExecutionContext, ankhFile *ankh.AnkhFile) string {
 
 	return *chart.Tag
 }
+
+func ReplaceFormatVariables(format string, chart string, version string, env string) (string, error) {
+
+	result := format
+	currentUser, err := user.Current()
+	if err != nil {
+		return "", err
+	}
+
+	// Replace %USER%
+	result = strings.Replace(result, "%USER%", currentUser.Username, 1)
+
+	// Replace %CHART%
+	result = strings.Replace(result, "%CHART%", chart, 1)
+
+	// Replace %VERSION%
+	result = strings.Replace(result, "%VERSION%", version, 1)
+
+	// Replace %TARGET%
+	result = strings.Replace(result, "%TARGET%", env, 1)
```

Comment:
shouldn't these actually pass -1 for N, since we want to replace all occurrences?